alacritty_terminal/Cargo.toml

```diff
@@ -49,4 +49,4 @@ windows-sys = { version = "0.36", features = [
 ]}
 
 [dev-dependencies]
-serde_json = "1.0.0"
+serde_json = "1.0.0"
```

Comment:
Missing newline.

Reply:
Fixed in https://github.com/alacritty/alacritty/pull/6896/commits/d067434b45ddb26b5dd181a98d6fc022962a9a51.